java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -309,13 +311,15 @@ public interface StringBaseCommands {
      * @see <a href="https://valkey.io/commands/msetnx/">valkey.io</a> for details.
      * @param keyValueMap A key-value map consisting of keys and their respective values to set.
      * @return <code>true</code> if all keys were set. <code>false</code> if no key was set.
+     * @param ArgType you can only pass Keys and Values of types {@link String} and {@link
```

Comment:
`ArgType` is not the param, the param is `keyValueMap`

Reply:
done